### PR TITLE
(feat) Added last attendee of visit in call patient modal

### PR DESCRIPTION
**Pull request by CynthiaKamau**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
Added last attendee of visit in call patient modal (patient or treatment supporter ) .This is configurable

## Screenshots
https://www.loom.com/share/d645b5fc3c814d9fb49c5602a569ba98

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-outpatient-app/src/config-schema.ts [hunk 1]
```diff
@@ -81,6 +81,10 @@ export const configSchema = {
       _type: Type.ConceptUuid,
       _default: '1343AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
     },
+    visitAttendeeConceptUuid: {
```
Comment (donaldkibet) [inline packages/esm-outpatient-app/src/config-schema.ts:84]: A description here would be helpful to in knowing what it does

File: packages/esm-outpatient-app/src/helpers/functions.ts [hunk 2]
```diff
@@ -58,3 +58,15 @@ export function generateVisitQueueNumber(service, maxVisitNumber) {
   const visitNumber = (maxVisitNumber + 1)?.toString().padStart(4, '0');
   return servicePrefix + '-' + visitNumber;
 }
+
+export function findObsByConceptUUID(arr, id) {
```
Comment (donaldkibet) [inline packages/esm-outpatient-app/src/helpers/functions.ts:62]: we could add types for `arr` and `id` to better understand what this function is doing

**Review by donaldkibet — CHANGES_REQUESTED** [review]
LGTM Thanks @CynthiaKamau left some suggestions

File: packages/esm-outpatient-app/src/config-schema.ts (hunk 1 above)
Comment (makombe) [reply packages/esm-outpatient-app/src/config-schema.ts:84]: @CynthiaKamau  Please make the implementation generic as we have discussed.

Comment (CynthiaKamau) [reply packages/esm-outpatient-app/src/config-schema.ts]: Done

**Comment by donaldkibet** [comment]
Please resolve this conflict

**Review by donaldkibet — APPROVED** [review]

**Pull request merged**